deepchem/models/torch_models/prot_bert.py

```diff
@@ -41,12 +41,13 @@ class ProtBERT(HuggingFaceModel):
     >>> from deepchem.models.torch_models.prot_bert import ProtBERT
     >>> pretrain_model_dir = os.path.join(tempdir, 'pretrain-model')
     >>> model_path = 'Rostlab/prot_bert'
-    >>> pretrain_model = ProtBERT(task='mlm', HG_model_path=model_path, n_tasks=1, model_dir=pretrain_model_dir)  # mlm pretraining
+    >>> pretrain_model = ProtBERT(task='mlm', model_pretrain_dataset='uniref100', n_tasks=1, model_dir=pretrain_model_dir)  # mlm pretraining
```

Comment:
I think this change makes it more rigid and less flexible. Let's revert to the previous design and add a comment explaining possible models